src/parser/core/Parser.js

```diff
@@ -122,6 +122,8 @@ class Parser {
 		})
 	}
 
+	_getDepHandle = (dep) => typeof dep === 'string'? dep : dep.handle
```

Comment:
If you `export` the `interface MappedDependency`, you can refer to it with:

```ts
/** @param {import('./Module').MappedDependency} dep */
```

Reply:
JS is JS. Will be resolved when porting `Parser`.